stablehlo/transforms/optimization/Passes.td

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2022 The StableHLO Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+include "mlir/Pass/PassBase.td"
+
+def StablehloAggressiveFolderPass
+    : Pass<"stablehlo-aggressive-folder", "func::FuncOp"> {
+  let summary = "Folds StableHLO operations";
+  let dependentDialects = [
+    "mlir::stablehlo::StablehloDialect",
+  ];
+  let options = [
+    Option<"foldFloat", "fold-float", "bool", /*default=*/"true",
+           "Allow for potentially lossy computations using float type.">,
+  ];
+}
+
+def StablehloAggressiveSimplificationPass
+    : Pass<"stablehlo-aggressive-simplification", "func::FuncOp"> {
+  let summary = "Canonicalizes StableHLO operations";
+  let dependentDialects = [
+    "mlir::stablehlo::StablehloDialect",
+    "mlir::arith::ArithDialect",
```

Comment:
[no action] seems to be missing in the original td file.

Reply:
Indeed it was. I noticed this as I was porting things.